format and clean up code, tweak gradient and remove page buttons on

adamkumlin opened the pull request:
small screens.

clicking on the small page buttons is challenging enough on a small screen. in my opinion it is better to just remove them entirely rather than making them even smaller. i am however open to suggestions and other opinions

closes #59

adamkumlin committed:
format and clean up code, tweak gradient and remove page buttons on small screens

simon-s-99 commented on style.css:
```diff
@@ -257,16 +262,15 @@ main div #enlarged-image {
     margin-right: auto;
     margin-left: auto;
     margin-bottom: 2rem;
 }
```
^ this is old code from when we were using the arrow icons (.svg)
feel free to remove :)

simon-s-99 commented on index.html:
```diff
@@ -24,111 +24,114 @@
+
+<div id="navigationButtons">
+  <button type="button" id="previous">
+    << /button>
```
remove this, it is causing this V

![image](https://github.com/adamkumlin/frontend-image-search/assets/142667693/f7987284-f30c-4995-9264-aa2bf13bf605)

adamkumlin replied:
the << /button> seems to be the culprit here, should be fixed now

simon-s-99 commented on style.css:
```diff
@@ -365,6 +373,10 @@ footer {
 
+    #pageButtons {
+        display: none;
+    }
```
Currently the numbered page buttons do not show up on the first page of desktop version, the problem is most likely  here or somewhere else in the css as i can see no major changes to the script.js file

![image](https://github.com/adamkumlin/frontend-image-search/assets/142667693/4820c629-21c6-4370-b250-7707bcc2266c)

adamkumlin replied:
same, see above

simon-s-99 reviewed (changes requested):
Overall i like the decision to remove the buttons on mobile, good call 👍 

Also, could we make the numbered page buttons just a little smaller on desktop? 🙂
![image](https://github.com/adamkumlin/frontend-image-search/assets/142667693/372dbce1-52f1-40d1-82eb-ecda28d45b53)

adamkumlin committed:
fix bugs

adamkumlin commented:
> Overall i like the decision to remove the buttons on mobile, good call 👍
> 
> Also, could we make the numbered page buttons just a little smaller on desktop? 🙂 ![image](https://private-user-images.githubusercontent.com/142667693/302902756-372dbce1-52f1-40d1-82eb-ecda28d45b53.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDczMDQzNjEsIm5iZiI6MTcwNzMwNDA2MSwicGF0aCI6Ii8xNDI2Njc2OTMvMzAyOTAyNzU2LTM3MmRiY2UxLTUyZjEtNDBkMS04MmViLWVjZGEyOGQ0NWI1My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwN1QxMTA3NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZjBhMTE0MTc5ODNhMGY4NmI3ODg1Njc5YWEyOGYzMjIxYTcwYjI4NjM2NjA4YzVlNjM3YzBmYmY1NWEyNmMyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.mgUVVdI5_UM6EG1ayN-JsHjv1Mi7x7UHs3wyeN7Utv4)

the << /button> seems to be the culprit here, should be fixed now

simon-s-99 committed:
bugfix: footer text on small screens

footer text is now working well on screens as small as 280 px wide

at 280 px wide the header text breaks but this width is not in popular use today

simon-s-99 commented:
good job 💯

simon-s-99 commented:
i miss-clicked and did not mean to close 🤣
